lms/djangoapps/discussion/rest_api/exceptions.py

```diff
@@ -17,3 +18,8 @@ class ThreadNotFoundError(ObjectDoesNotExist):
 class CommentNotFoundError(ObjectDoesNotExist):
     """ Comment was not found. """
     pass
+
+class DiscussionBlackedOut(APIException):
+    """ Discussion in black out period. """
+    status_code = 403
+    default_detail = 'Discussion are in black out period.'
```

Comment:
`Discussion are`?

`blackout`

Reply:
this message can be improved.

Reply:
we can consult the mobile team on what would be a better message for them!